source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +234,89 @@ export default class WindowManager {
       return
     }
 
+    this._mainWindow.on('show', () => {
+      const leaveAppRunning = Boolean(global.config.get('system.leaveAppRunning'))
+      if (leaveAppRunning) {
```

Comment:
This is good. It implements #4:
> If Zettlr is running AND the `Show app in the notification area` is true, the tray icon will be visible (regardless if Zettlr window(s) are visible or not).

In branch `issue175` the tray is in a function. So the above would become:
``` typescript
    this._mainWindow.on('show', () => {
      const leaveAppRunning = Boolean(global.config.get('system.leaveAppRunning'))
      if (leaveAppRunning) {
        this._addTray()
```